JS101/09-javascript-dev-tools.md

```diff
@@ -0,0 +1,10 @@
+this lesson is all about what you can do with chrome's devtools to debug and work with Javascript
```

Comment:
someone else just hacked this one together... but it's not showing as a conflict because you've renamed the files.  Go ahead and pull from master then rename the new one 😄